webhooks-extension/src/components/WebhookCreate/WebhookCreate.js

```diff
@@ -23,7 +23,7 @@ function validateInputs(value, id) {
       return false;
     }
 
-    if (/[^-.a-z1-9]/.test(trimmed)) {
+    if (/[^-a-z0-9]/.test(trimmed)) {
```

Comment:
is there a reason we don't use the same regex as kubernetes?

Reply:
so this regex will return true if the `trimmed` value contains any invalid character while the one in Kubernetes will return true if the variable contains only valid characters. Its just the opposite but i guess we use the first simply because it is more compact and simple 😅 (same one is used in the secrets page).

Reply:
the regex provided by kubernetes does more than just ensure it contains valid characters, it enforces all of the rules including the fact that it can only start/end with an alphanumeric character (not `-` or `.`)

![image](https://user-images.githubusercontent.com/2829095/67576883-5c945e00-f737-11e9-8fcf-ca2c2bf0084c.png)